lib/bemxjst/index.js

```diff
@@ -20,8 +19,8 @@ function BEMHTML(options) {
   this.match = null;
 
   // Create new Context constructor for overriding prototype
-  this.contextConstructor = function ContextChild(bemhtml) {
-    Context.call(this, bemhtml);
+  this.contextConstructor = function ContextChild(bemxjst) {
+    Context.call(this, bemxjst);
```

Comment:
I think there might be reason to inherit it from `this.Context`, as in future we may like to have two different kinds of `Context` for BEMTREE and BEMHTML.